### ISSUE TITLE
Cannot install with upgraded dashboard

### ISSUE THREAD

**Issue by EnnPeeTee** [patrickknabe/node-red-contrib-ui-clock#2]
Hi,

Whilst attempting to upgrade Dashboard from 2.30 to 3.1.2, I discovered node-red-contrib-ui-clock was causing issues. After deleting this node, the upgrade was fine.

However, I am unable to reinstall this truly delightful node and it would seem there are dependency issues. 

Any ideas, please?

All advice gratefully received!

**Comment by colinl** [comment]
The error is
```
$ npm install node-red-contrib-ui-clock
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: tigger@1.0.0
npm ERR! Found: node-red-dashboard@3.1.2
npm ERR! node_modules/node-red-dashboard
npm ERR!   node-red-dashboard@"~3.1.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer node-red-dashboard@"^2.23.2" from node-red-contrib-ui-clock@1.0.1
npm ERR! node_modules/node-red-contrib-ui-clock
npm ERR!   node-red-contrib-ui-clock@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```
Adding --legacy-peer-deps does allow it to install.  
This is using npm 8.1.0.

**Comment by patrickknabe** [comment]
Hi,

I just corrected the peerDependency for the Dashboard and released a new version of the clock (v1.0.2). Should be already available.

Maybe you can give me a short feedback?

Have fun with it and best regards
Patrick

**Comment by EnnPeeTee** [comment]
Hi Patrick,

 

Thank you so much for the update.  Your kind use of your time to create nodes and generous sharing of your achievement is really appreciated!

 

All is now updated and working well!

 

I am very new to Node-Red and find it quite exciting with limitless opportunities for the future.  I am currently in the process of creating an on-board system management service for my small boat using a Raspberry Pi running Node-Red with the intention of monitoring engine temperatures and pressures and providing navigation displays from a GPS source.  One thing I need to determine is whether one can customise the display of ‘gauges’, like your excellent clock. I wonder if one can superimpose a node on top of a graphic to visually embellish the display.

 

 

Cheers and many thanks again,

 

 

Nigel
